docs/index.bs

```diff
@@ -364,6 +366,7 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
             1. Else, let it be initialized to a new {{Client}} object that represents the worker associated with |incumbentGlobal|.
             1. Let the {{ExtendableMessageEvent/ports}} attribute of |e| be initialized to |newPorts|.
             1. <a>Dispatch</a> |e| at |destination|.
+            1. Invoke [=Update Service Worker Extended Events Set=] with |serviceWorker| and |e|.
```

Comment:
We gate all the service worker events through Handle Fetch, Handle Foreign Fetch, Handle Functional Events, and ServiceWorker.postMessage(). So, maintenance would be okay. Also, I intended it to be called after dispatching an event returns.